### DISCUSSION TITLE
Cannot use `@google-cloud/vision` library in extension

### DISCUSSION THREAD

**Discussion by keer4n** [directus/directus#14864] [Archive]
### Describe the Bug

I am trying to build an extension which uses `@google-cloud/vision` library, the extension builds fine with `@directus/extensions-sdk 9.14.4` but when trying to load in directus, shows this error:
<img width="500" alt="Screen Shot 2022-08-04 at 3 45 24 PM" src="https://user-images.githubusercontent.com/8868294/182958385-40277e8e-5f6b-4fa0-a578-f810ecc4547b.png">
Tried it with `@directus/extensions-sdk 9.4.0` as mentioned in #11275, no luck. 

### To Reproduce

Just including the library in extension src.
```js
import { defineHook } from "@directus/extensions-sdk";
import vision from "@google-cloud/vision";

export default defineHook(({ filter, action }) => {
  filter("items.create", () => {
    const client = new vision.ImageAnnotatorClient();
    console.log("Creating Item!");
  });

  action("items.create", () => {
    console.log("Item created!");
  });
});
```

### Errors Shown

<img width="500" alt="Screen Shot 2022-08-04 at 3 45 24 PM" src="https://user-images.githubusercontent.com/8868294/182958738-416cf2f1-183d-4b5d-86ff-fbf0c80c5119.png">

### What version of Directus are you using?

9.14.1

### What version of Node.js are you using?

v16.14.2

### What database are you using?

Postgres

### What browser are you using?

any

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
Looks very similar to https://github.com/googleapis/nodejs-vision/issues/240 to me. I think this might be a compatibility problem of the library with bundlers, rather than a bug in directus 🤔
